CMakeLists.txt

```diff
@@ -37,6 +39,17 @@ if (NOT fmt_FOUND)
     message(DEBUG "Non CMake package for fmt found. fmt was retrieved via FetchContent.")
 endif ()
 
+if (NOT expected-lite_FOUND)
```

Comment:
Need some help here cannot figure out the correct cmake magic to make it work 😅

Reply:
see https://github.com/rdf4cpp/rdf4cpp/pull/66/commits/9ca09f45cc3ae8d5d55863fe00a5c3497a07ca3c

You normally add only your own headers to `target_include_directories`. CMake takes care that  headers of dependencies are visible within the library and within the interface if required. 

If assets from dependencies are exposed via the public headers of our library, the dependencies must be in the `PUBLIC` section of `target_link_libraries`.